gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -273,6 +277,9 @@ public void execute(Task _task) {
                     jarTask.get().getArchiveFile().get(), p.getConfigurations().getByName("runtimeClasspath")));
             task.setArgs(distributionExtension.getArgs().get());
             task.setJvmArgs(ImmutableList.builder()
+                    .addAll(javaAgentConfiguration.getFiles().stream()
```

Comment:
this will resolve the configuration at configuration time, which will make GCV sad. We will need to set this slightly differently. I believe the task exposes a way for you to register an args provider which is only evaluated at execution time

Reply:
Just discovered this testing with log-receiver -- I've added a JavaExec doFirst action which appears to do what we want. lmk if there's a smarter way to go about it though.

Reply:
let me open this up in Intellij and maybe I can find something!